utils/wrappers/kyberNetworkWrapper.ts

```diff
@@ -0,0 +1,297 @@
+import * as _ from 'lodash';
+import { Address } from 'set-protocol-utils';
+import { BigNumber } from 'bignumber.js';
+
+import { StandardTokenMockContract } from '@utils/contracts';
+
+import { getWeb3 } from '../web3Helper';
+import { ether } from '@utils/units';
+
+import { UNLIMITED_ALLOWANCE_IN_BASE_UNITS, DEFAULT_GAS } from '../constants';
+
+import { ConversionRateABI } from '../external/abis/ConversionRateABI';
+import { KyberNetworkABI } from '../external/abis/KyberNetworkABI';
+import { KyberReserveABI } from '../external/abis/KyberReserveABI';
+
+import { KYBER_CONTRACTS, KYBER_PERMISSIONED_ACCOUNTS } from '../kyberSnapshotAddresses';
+
+const web3 = getWeb3();
+
+
+export class KyberNetworkWrapper {
```

Comment:
Nit but can we call this and the file as a whole `KyberNetworkConfigWrapper`? Or something similar. Since we have a contract of the same name it's just easier to differentiate.

Reply:
+1 I think we should start just calling these `Helper` classes. The wrapper suffix is confusing in general. Was clear when I was working in the repo a lot but now in review it causes slight context hiccups

Reply:
Ok - I can create a trello card to do the overall refactor.